README.md

```diff
@@ -19,28 +19,32 @@ This crate is `#[no_std]` compatible with `default-features = false`
 | Feature                | Default? | Description |
 | :---                   | :---     | :---        |
 | `alloc`                | ✓        | Enables features that require dynamic heap allocation |
-| `std`                  | ✓        | std::error::Error types |
-| `zeroize`              | ✓        | Enables `Zeroize` for `SigningKey` |
-| `asm`                  |          | Assembly implementation of SHA-2 compression functions |
-| `batch`                |          | Batch verification. Requires `alloc` |
-| `digest`               |          | TODO |
-| `legacy_compatibility` |          | See: A Note on Signature Malleability |
-| `pkcs8`                |          | PKCS#8 Support |
-| `pem`                  |          | PEM Support |
-| `rand_core`            |          | TODO        |
+| `std`                  | ✓        | Implements `std::error::Error` for `SignatureError` |
+| `zeroize`              | ✓        | Implements `Zeroize` and `ZeroizeOnDrop` for `SigningKey` |
+| `rand_core`            |          | Enables `SigningKey::generate` |
+| `batch`                |          | Enables `verify_batch` for verifying many signatures quickly. Also enables `rand_core`. |
+| `digest`               |          | Enables `Context`, `SigningKey::{with_context, sign_prehashed}` and `VerifyingKey::{with_context, verify_prehashed, verify_prehashed_strict}` for Ed25519ph prehashed signatures |
+| `asm`                  |          | Enables assembly optimizations in the SHA-512 compression functions |
+| `pkcs8`                |          | Enables [PKCS#8](https://en.wikipedia.org/wiki/PKCS_8) serialization/deserialization for `SigningKey` and `VerifyingKey` |
+| `pem`                  |          | TODO |
```

Comment:
@tarcieri  can you fill this and `pkcs8` out? I'm pretty confused actually. How is `pem` different from PKCS#8? Isn't PEM a container format within the PKCS#8 spec? What does it contain? What are the other things in PKCS#8?

Reply:
PEM "Privacy Enchanced Email" (yes confusing) is a text file format .. like Base64 wrapping/formatting of binary X.509 certificates e.g. PEM is the --BEGIN ASCII blocks that simply contain the Base64 formatting of the PKCS#8
The origin was from the early internet as e-mail in it's 7bit form wasn't really designed to carry binary data and it was encoded and wrapped (the --BEGIN block) in as Base64 text formatting that allowed itself to be carried in 7bit SMTP - https://en.wikipedia.org/wiki/Base64
PKCS#8 in turn is the binary encoded container under it which may contain passphrase etc - it's just ASN.1/BER (or was it DER) whilst the PEM is the text Base64 wrapping on top of it
PKCS#8 ASN.1 schema is here - https://datatracker.ietf.org/doc/html/rfc5208#page-5